lifecycle_msgs/msg/State.msg

```diff
@@ -0,0 +1,23 @@
+# Primary state definitions
+# as depicted in
+# design.ros2.org
+
+uint8 PRIMARY_STATE_UNKNOWN = 0
+uint8 PRIMARY_STATE_UNCONFIGURED = 1
+uint8 PRIMARY_STATE_INACTIVE = 2
+uint8 PRIMARY_STATE_ACTIVE = 3
+uint8 PRIMARY_STATE_FINALIZED = 4
+
+# Transition state definitions
+# as depicted in
+# design.ros2.org
+
+uint8 TRANSITION_STATE_CONFIGURING = 10
+uint8 TRANSITION_STATE_CLEANINGUP = 11
+uint8 TRANSITION_STATE_SHUTTINGDOWN = 12
+uint8 TRANSITION_STATE_ACTIVATING = 13
+uint8 TRANSITION_STATE_DEACTIVATING = 14
+uint8 TRANSITION_STATE_ERRORPROCESSING = 15
```

Comment:
We don't align those since that implies if you ever add another line which is longer you have to touch all other lines.